### PR TITLE
Refactor Title 1's subtitle data structure

### LINKED ISSUE #64: [TASK] Refactor Title I Json document
The current json for Title 1 has a misplaced data. The program should be subtitle, subprogram should be program. Create a code to generate the output with the correct structural order.

### PR DESCRIPTION
**Pull request by ywkim312**
@sandeep-ps Please let me know if the output Jsons look okay.

### CODE REVIEW

File: title-1-commodities/dmc_subprograms_data.json
Comment (pengyin-shan) [inline title-1-commodities/dmc_subprograms_data.json:1]: for consistency, can we still have the 'programs' attribute but return an empty array for subtitles with no sub-programs? This seems to be what the program did for subprograms. If a program has no subprogram, the 'subprogram' attribute will return an empty array.

Comment (ywkim312) [reply title-1-commodities/dmc_subprograms_data.json:1]: so you want to add something like this?
```
"programs":[]
```
or
```
"programs":[
    "subPrograms": []
]
```
I assume it is the first one.

Comment (pengyin-shan) [reply title-1-commodities/dmc_subprograms_data.json:1]: the first option would be good. If the front-end detected that the 'programs' return an empty array, then there's no need to check subprograms.

Comment (ywkim312) [reply title-1-commodities/dmc_subprograms_data.json:1]: Updated, please check it out

**Review by pengyin-shan — APPROVED** [review]

File: title-1-commodities/dmc_subprograms_data.json
Comment (pengyin-shan) [reply title-1-commodities/dmc_subprograms_data.json:1]: awesome, looks good to me!

File: title-1-commodities/dmc_subprograms_data.json [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
-      "paymentInDollars": 1969315257.26,
-      "totalCounts": 1259532,
-      "averageRecipientCount": 157441.5,
-      "subPrograms": []
-    }
-  ]
+  "subtitleName": "Dairy Margin Coverage, Subtitle D",
+  "subtitlePaymentInDollars": 1969315257.26,
+  "totalCounts": 1259532,
+  "averageRecipientCount": 157441.5
```
Comment (sandeep-ps) [inline title-1-commodities/dmc_subprograms_data.json]: Here, if needed, `"programs": []` could be added for consistency. Other details look good to me. I will approve. Thanks.

**Review by sandeep-ps — APPROVED** [review]
Approving. Thanks.

File: title-1-commodities/dmc_subprograms_data.json (hunk 1 above)
Comment (ywkim312) [reply title-1-commodities/dmc_subprograms_data.json]: add "programs": []

**Pull request merged**